examples/vm_os_upgrade/06_reboot_cloned_vm.yml

```diff
@@ -0,0 +1,8 @@
+- name: Reboot VM
+  scale_computing.hypercore.vm_params:
+    vm_name: "{{ cloned_vm_name }}"
+    power_state: reboot
+
+- name: Pause for 20 seconds, so that VM becomes reachable
+  ansible.builtin.pause:
+    seconds: 20
```

Comment:
reachable in which sense? Ansible can SSH into VM?

Reply:
Yes, reachable via SSH, I updated the task name